types/src/crypto_proxies.rs

```diff
@@ -57,6 +57,14 @@ impl<Sig: RawSignature> SignatureWrapper<Sig> {
     ) {
         li_with_sig.add_signature(author, self.0)
     }
+
+    pub fn remove_from_li(
```

Comment:
Why do we need this function? Is it a leftover from some previous experiments?

Reply:
Good catch! Apologies, was only focusing on core logic and didn;t sanitize before posting PR. Removed.